node/pkg/driver/node_utils.go

```diff
@@ -347,3 +348,10 @@ func (n NodeUtils) IsNotMountPoint(file string) (bool, error) {
 func (n NodeUtils) GetPodPath(origPath string) string {
 	return path.Join(PrefixChrootOfHostRoot, origPath)
 }
+
+func (n NodeUtils) ShortenNodeID(nodeId string) string {
+	fcDelimiter := ":"
```

Comment:
all these chars should be consts, to avoid this duplication

Reply:
not relevant